<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inicio</class>
 <widget class="QMainWindow" name="Inicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controle Financeiro</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>420</y>
       <width>691</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget {
background: none;
background-color: #B9B7B7;
border-radius: 15;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="btt_entrada">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
color: #fff;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="btt_saida">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
color: #fff;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="btt_total">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>70</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
color: #fff;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>250</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #fff;
border-radius: 15;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>250</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #fff;
border-radius: 15;</string>
     </property>
    </widget>
    <widget class="QLabel" name="btt_entrada_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
color: #fff;
font: bold;</string>
     </property>
     <property name="text">
      <string>Descrição</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="btt_entrada_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>220</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
color: #fff;
font: bold;</string>
     </property>
     <property name="text">
      <string>Valor</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="Entrada">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>240</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
border-radius: 15;</string>
     </property>
     <property name="text">
      <string>Entrada</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Saida">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>270</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
border-radius: 15;</string>
     </property>
     <property name="text">
      <string>Saída</string>
     </property>
    </widget>
      <widget class="AnimatedButton" name="pushButton">
    <property name="geometry">
        <rect>
            <x>590</x>
            <y>230</y>
            <width>131</width>
            <height>61</height>
        </rect>
    </property>
    <property name="font">
        <font>
            <pointsize>16</pointsize>
        </font>
    </property>
    <property name="styleSheet">
        <string notr="true">
            AnimatedButton {
                border-radius: 15px;
                background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0,
                                                stop:0 #ff8c00, stop:0.5 #ffa500, stop:1 #ffbf00);
                border: none;
                font: bold;
                color: #fff;  /* Cor da fonte branca */
            }

            AnimatedButton:hover {
                background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0,
                                                stop:0 #ffbf00, stop:0.5 #ffa500, stop:1 #ff8c00);
            }
        </string>
    </property>
    <property name="text">
        <string>ADICIONAR</string>
    </property>
      </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
color: #fff;</string>
     </property>
     <property name="text">
      <string>Entradas</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
color: #fff;</string>
     </property>
     <property name="text">
      <string>Saídas</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>30</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
color: #fff;</string>
     </property>
     <property name="text">
      <string>Total</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>811</width>
       <height>601</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/img/pag2.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <zorder>frame</zorder>
    <zorder>tableWidget</zorder>
    <zorder>btt_entrada</zorder>
    <zorder>btt_saida</zorder>
    <zorder>btt_total</zorder>
    <zorder>lineEdit</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>btt_entrada_2</zorder>
    <zorder>btt_entrada_3</zorder>
    <zorder>Entrada</zorder>
    <zorder>Saida</zorder>
    <zorder>pushButton</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimatedButton</class>
   <extends>QPushButton</extends>
   <header location="global">animated_button</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="design1.qrc"/>
 </resources>
 <connections/>
</ui>
